src/daemon/daemon.cpp

```diff
@@ -3315,37 +3344,62 @@ mp::MountHandler::UPtr mp::Daemon::make_mount(VirtualMachine* vm, const std::str
                : vm->make_native_mount_handler(config->ssh_key_provider.get(), target, mount);
 }
 
-void mp::Daemon::configure_new_interfaces(const std::string& name, mp::VirtualMachine& vm, mp::VMSpecs& specs)
+// This function configures the network interfaces whose MAC address is empty. They will stay in specs.extra_interfaces
+// only if the backend is able to add them to the virtual machine. If not, they will be removed.
+std::unordered_set<std::string> mp::Daemon::configure_new_interfaces(const std::string& name,
```

Comment:
I think the `std::unordered_set<std::string>` return type should also be commented to indicated it's a set of interfaces that could not be configured and added.  Otherwise, without really reading the code, it would seem it would be a set of the interfaces that were configured.

Reply:
Right, done.